src/authenticated/project/ProjectSection.js

```diff
@@ -46,7 +46,7 @@ const ProjectSection = ({ project, editing, onFormDataChange }) => {
         label="Add Champion"
         {...commonProps}
       />
-
+    
```

Comment:
This is a weird change. I would have thought that prettier would have removed trailing whitespace

Reply:
Yes it should have.  It looks like I must have committed the changes without installing the modules first?  I fixed this, but I wonder if there is a way to prevent this from happening?